profiles/base.yaml

```diff
@@ -608,3 +608,8 @@ discard_scrolls:
 - Seal Cambrinth
 - Strange Arrow
 
+theurgy_whitelist:
+theurgy_blacklist:
+- embarass_myself
+theurgy_exp_threshold: 0
```

Comment:
Should this default to `31`? I'm looking at how it's used: https://github.com/rpherbig/dr-scripts/pull/2035/files#diff-22f011c13b942020ec0c0acc7e0be846R703

Reply:
I just simplified this logic a bunch -- I think the scholarship exp threshold was overkill, and likely never to be used.  There are now two modes: `theurgy_exp_threshold <= 0`, which means "run all rituals", or `theurgy_exp_threshold > 0`, which means "run until threshold is reached.

Reply:
This doesn't seem very clear to me in the code.